src/app/visual/inputs/inputs-demo.component.html

```diff
@@ -0,0 +1,38 @@
+<div style="width: 400px">
```

Comment:
It would probably be more meaningful to have the native input and the SKY UX version side by side, for visual comparison (to make sure the native input looks good with and without the CSS class).

You may need to break the table up into parts in order to get appropriate screenshots.

```
<table class="sky-inputs-demo-table">
  <thead>
    <tr>
      <th>Type</th>
      <th>Native</th>
      <th>SKY UX</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>button</td>
      <td><input type="button" value="Click me"></td>
      <td><input type="button" value="Click me" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>checkbox</td>
      <td><input type="checkbox"></td>
      <td><input type="checkbox" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>radio</td>
      <td><input type="radio"></td>
      <td><input type="radio" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>color</td>
      <td><input type="color"></td>
      <td><input type="color" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>date</td>
      <td><input type="date"></td>
      <td><input type="date" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>datetime-local</td>
      <td><input type="datetime-local"></td>
      <td><input type="datetime-local" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>email</td>
      <td><input type="email"></td>
      <td><input type="email" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>file</td>
      <td><input type="file"></td>
      <td><input type="file" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>hidden</td>
      <td><input type="hidden"></td>
      <td><input type="hidden" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>image</td>
      <td><input type="image" [src]="imageData"></td>
      <td><input type="image" [src]="imageData" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>month</td>
      <td><input type="month"></td>
      <td><input type="month" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>number</td>
      <td><input type="number"></td>
      <td><input type="number" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>password</td>
      <td><input type="password"></td>
      <td><input type="password" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>range</td>
      <td><input type="range"></td>
      <td><input type="range" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>reset</td>
      <td><input type="reset"></td>
      <td><input type="reset" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>submit</td>
      <td><input type="submit"></td>
      <td><input type="submit" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>tel</td>
      <td><input type="tel"></td>
      <td><input type="tel" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>text</td>
      <td><input type="text"></td>
      <td><input type="text" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>time</td>
      <td><input type="time"></td>
      <td><input type="time" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>url</td>
      <td><input type="url"></td>
      <td><input type="url" class="sky-form-control"></td>
    </tr>
    <tr>
      <td>week</td>
      <td><input type="week"></td>
      <td><input type="week" class="sky-form-control"></td>
    </tr>
  </tbody>
</table>

```